### DISCUSSION TITLE
Update 2022-10-20

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#147] [Announcements]
Implemented a test of custom motion inspector for kons-9 data. I am inspired by the ideas of Glamorous Toolkit, and want to base our GUI on it.

https://gtoolkit.com

The new inspector is an outliner which depicts the timing info of the scene's motions:

https://user-images.githubusercontent.com/103860793/197078264-874e1b4f-d52a-4973-a477-ce3827cf2c1d.mov

**Comment by kaveh808** [comment]
Output to Blender.

OBJ format:
![Screen Shot 2022-10-20 at 2 46 41 PM](https://user-images.githubusercontent.com/103860793/197086015-93c36e06-efe0-4b4d-b9c5-b9a58b65eb7b.png)

USD format:
![Screen Shot 2022-10-20 at 2 48 17 PM](https://user-images.githubusercontent.com/103860793/197086068-35f80668-18e8-435a-93bd-9e227330b0d6.png)

**Comment by jolby** [comment]
Incredible!

**Comment by lukego** [comment]
cc @girba
